modules/ROOT/pages/pricing-metrics.adoc

```diff
@@ -0,0 +1,138 @@
+= Pricing and Usage Metrics
+
+[In progress]
+
+MuleSoft offers a pricing and packaging model for Anypoint Platform that allots a number of Mule flows, Mule messages, and data throughput (measured as network I/O bytes). Not all types of Mule flows and messages count towards the allotments in a package. 
+
+[[mule-flows]]
+== Mule Flows
+
+A _Mule flow_ is a sequence of logical operations configured in XML using the `<flow>` element. A Mule app can consist of a single Mule flow, or it can break up processing into discrete xref:mule-runtime::about-flows.adoc[Mule Flows and subflows] that you add to the app and connect together. Mule apps in production environments typically use multiple Mule Flows and subflows to divide the app into functional modules or for error-handling purposes. For example, one Mule flow might receive a record and transform data into a given format that another flow processes in some special way. 
```

Comment:
```suggestion
A _Mule flow_ is a sequence of logical operations configured in XML by using the `<flow>` element. A Mule app can consist of a single Mule flow, or it can break up processing into discrete xref:mule-runtime::about-flows.adoc[Mule flows and subflows] that you add to the app and connect together. Mule apps in production environments typically use multiple Mule flows and subflows to divide the app into functional modules or for error-handling purposes. For example, one Mule flow might receive a record and transform data into a given format that another flow processes in some special way. 
```